### PR TITLE
Github #36 storing image in moodledata instead of db.

### PR DESCRIPTION
**Pull request by jainkavita31**

### CODE REVIEW

File: ajax.php [hunk 1]
```diff
@@ -44,7 +44,8 @@
     // If it is not main image, get the main image data and compare.
     if ($mainentry = $DB->get_record('quizaccess_proctor_data',
         array('userid' => $USER->id, 'quizid' => $cm->instance, 'image_status' => 'M', 'attemptid' => $attemptid))) {
-        $target = $mainentry->userimg;
+        $target = $mainentry->userimg;        
+        $target = file_get_contents("$CFG->dataroot.'/temp/quizaccess_quizproctoring/captured/'.$mainentry->userimg");
```
Comment (msoni11) [inline ajax.php:48]: What's the purpose of ```$target = $mainentry->userimg;```  line

Comment (msoni11) [reply ajax.php:48]: What's temp here?

File: lib.php [hunk 2]
```diff
@@ -113,14 +119,19 @@ function quizproctoring_storeimage($data, $cmid, $attemptid, $quizid, $mainimage
...
+        $imagename = $id. "_" . $attemptid . "_" . $USER->id . '_image.png';
+        $proctoreddata = $DB->get_record('quizaccess_proctor_data', array('id' => $id));
+        $proctoreddata->userimg = $imagename;
+        $DB->update_record('quizaccess_proctor_data', $proctoreddata);
+    }
     $tmpdir = make_temp_directory('quizaccess_quizproctoring/captured/');
-    file_put_contents($tmpdir . 'myimage.png', $data);
-    $fs = get_file_storage();
+    file_put_contents($tmpdir . $imagename, $data);
+    /*$fs = get_file_storage();
```
Comment (msoni11) [inline lib.php:134]: If code is not required, delete it. Don't comment it.

File: ajax.php (hunk 1 above)
Comment (jainkavita31) [reply ajax.php:48]: temp directory in moodledata..

Comment (msoni11) [reply ajax.php:47]: https://moodledev.io/docs/apis/subsystems/files

**Pull request merged**